beekeeper-path-scheduler-apiary/src/main/java/com/expediagroup/beekeeper/scheduler/apiary/BeekeeperPathSchedulerApiary.java

```diff
@@ -52,7 +52,7 @@ public static void main(String[] args) {
   public void setApplicationContext(ApplicationContext applicationContext) throws BeansException {
     context = (ConfigurableApplicationContext) applicationContext;
     runner = (PathSchedulerApiaryRunner) context.getBean("pathSchedulerApiaryRunner");
-    meterRegistry = (MeterRegistry) context.getBean("registry");
+    meterRegistry = (MeterRegistry) context.getBean("graphiteMeterRegistry");
```

Comment:
Ah, OK, so that's what had to change. Up to you whether you prefer the method named "registry" or this. The former is more generic downstream - especially since you're casting it to the parent class here so theoretically you could change the type in the Context and everything would still work here as long as the method was still called "registry". If we name the method after the type it's more "correct" in the Context but does mean if we change the type there and give it a more descriptive method name then we need to change it here too.

Reply:
Will go for the compromise of "meterRegistry" -> that's probably the most descriptive.

Reply:
Even better yes!